content/en/docs/concepts/configuration/configmap.md

```diff
@@ -0,0 +1,168 @@
+---
+title: ConfigMaps
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< glossary_definition term_id="configmap" prepend="A ConfigMap is" length="all" >}}
+
+{{< caution >}}
+ConfigMap does not provide secrecy or encryption.
+
+If the data you want to store are confidential, use a
+{{< glossary_tooltip text="Secret" term_id="secret" >}} rather than a ConfigMap,
+or use additional (third party) tools to keep your data private.
+{{< /caution >}}
+
+{{% /capture %}}
+
+{{% capture body %}}
+## Motivation
+
+You can use a ConfigMap when you want to set configuration data for your workload that
+is separate from application code. For example: if you have code component that you can
```

Comment:
```suggestion
For example, storing different database locations for a code component that you can```

Reply:
@sftim, I think this example could be reworded (or you could generalize the example):
I think you are saying: you can use ConfigMaps to set different environments for running an application (development) and cloud (data center, on prem).
For example: if you are developing an application [or code component] to run in the cloud (...)

Reply:
Reworded (thanks @celestehorgan).